src/semgrep_agent/semgrep_app.py

```diff
@@ -26,7 +26,12 @@
 
 # 4, 8, 16 seconds
 RETRYING_ADAPTER = requests.adapters.HTTPAdapter(
-    max_retries=Retry(total=3, backoff_factor=4, method_whitelist=["GET", "POST"])
+    max_retries=Retry(
+        total=3,
+        backoff_factor=4,
+        method_whitelist=["GET", "POST"],
+        status_forcelist=(500, 502, 503),
```

Comment:
```suggestion
        status_forcelist=(413, 429, 500, 502, 503),
```

This readds two default retried statuses